app/assets/stylesheets/forms.scss

```diff
@@ -24,15 +24,15 @@ form label {
   }
 }
 
-input[type='text'] {
+input[type='text']:not(.govuk-input, .autocomplete__input) {
   @extend .form-control; // scss-lint:disable PlaceholderInExtend
 }
 
 .errors input {
   border-color: $state-danger-border;
 }
 
-button {
+button:not(.govuk-button) {
```

Comment:
I've solved this problem on My PR #665 by not including any of the bootstrap stuff. If my PR gets merged in before this then we won't be needing these changes

Reply:
Have now removed this commit.